### PR TITLE
Back to pascal case for readability.

### PR DESCRIPTION
**Pull request by Brilator**
Note: for the time being, the KB should be reachable under https://nfdi4plants.org/content/docs/ResearchDataManagement.html again. So no need to change links until KB is out-sourced. y

### CODE REVIEW

File: src/content/docs/AnnotatedResearchContext.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Name | Type | Tasks on metadata
 **[DataPLANT's QuickStart on ARCs][quickstart-arc]**   | Document | **Tutorial:** <ul><li>How to initialize an ARC<li>How to package data with metadata</li></ul>
 **[Swate][Swate]** <br> (Swate Workflow Annotation Tool for Excel) | Tool | **Collect and structure:** <ul><li>Annotate experimental and computational workflows with ISA metadata schema</li><li>Easy use of ontologies and controlled vocabularies</li><li>Metadata templates for versatile data types</li></ul>
 **[ArcCommander][ArcCommander]** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
-**[DataHUB](datahub.html)** | Service | **Share:** <ul><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
+**[DataHUB](DataHub.html)** | Service | **Share:** <ul><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
```
Comment (Freymaurer) [inline src/content/docs/AnnotatedResearchContext.md:85]: Please change `datahub.md` to `DataHub.md` for consistency

File: src/content/docs/AnnotatedResearchContext.md [hunk 2]
```diff
@@ -105,13 +105,13 @@ Name | Type | Tasks on metadata
...
-[kb-datahub]: ./datahub.html "DataPLANT DataHUB"
-[kb-FAIR]: ./fair.html "FAIR Data principles"
-[quickstart-arc]: ./quickstart_arc.html "Quickstart ARC"
-[kb-Repositories]: ./repositories.html "Repositories"
-[kb-RDM]: ./rdm.html "Research Data Management"
-[kb-DataSharing]: ./datasharing.html "Data Sharing"
+[kb-git]: ./VersionControlGit.html "Git"
+[kb-datahub]: ./DataHub.html "DataPLANT DataHUB"
+[kb-FAIR]: ./FairDataPrinciples.html "FAIR Data principles"
+[quickstart-arc]: ./QuickStart_arc.html "Quickstart ARC"
```
Comment (Freymaurer) [inline src/content/docs/AnnotatedResearchContext.md:111]: Please adjust `quickstart_arc.md` for consistency.

File: src/content/docs/DataPublications.md [hunk 3]
```diff
@@ -75,10 +75,10 @@ Name | Type | Tasks on metadata
 
 <!-- Knowledge base cross-references -->
 
-[kb-Metadata]: ./metadata.html "Metadata"
-[kb-pid]: ./pids.html  "Persistent Identifiers"
-[kb-datahub]: ./datahub.html "DataPLANT DataHUB"
-[kb-Repositories]: ./repositories.html "Repositories"
+[kb-Metadata]: ./Metadata.html "Metadata"
```
Comment (Freymaurer) [inline src/content/docs/DataPublications.md:78]: Please adjust `metadata.md` for consistency.

**Review by Freymaurer — CHANGES_REQUESTED** [review]

**Comment by Brilator** [comment]
No idea, why the file name changes did not make it to GitHub. They're all updated on my machine..

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**